site/source/docs/api_reference/proxying.h.rst

```diff
@@ -0,0 +1,144 @@
+.. _proxying-h:
+
+==========
+proxying.h
+==========
+
+The Emscripten `proxying.h
+<https://github.com/emscripten-core/emscripten/blob/main/system/include/emscripten/proxying.h>`_
```

Comment:
You can mode the long URL to the end of the file as:

```
.. _proxying.h: https://github.com/emscripten-core/emscripten/blob/main/system/include/emscripten/proxying.h
```

And then reference it as `\`preamble.js\``_`